packages/components/input/src/input.ts

```diff
@@ -166,6 +166,13 @@ export const inputProps = buildProps({
     type: definePropType<StyleValue>([Object, Array, String]),
     default: () => mutable({} as const),
   },
+  /**
+   * @description native input autofocus
+   */
+  autofocus: {
+    type: Boolean,
+    default: false,
```

Comment:
The default value here may be omitted.

Reply:
I saw other boolean props have default value, so i just want them keep same.